app/views/contact/govuk/new.html.erb

```diff
@@ -1,4 +1,13 @@
-<% content_for :title do "Contact GOV.UK" end %>
+<%
+  headings = {
+    title: "Contact GOV.UK",
```

Comment:
Does the title need including here? It doesn't look like one of the parts of the form that could have an error. I see you're using it on line 39 to avoid duplication of text - could it be separate variable if it's not needed for the error checking?